backend/agents/persona_agent.py

```
@@ -162,56 +483,136 @@ def _get_system_prompt(self) -> str:
+        # so it's available when tools are called during execution
+        if self.vectorstore:
+            try:
+                self.vectorstore.add_texts(
+                    [f"User: {message}"],
+                    metadatas=[{
+                        "persona_id": str(self.persona.id),
+                        "context_type": "conversation",
+                        "message_type": "user",
+                        "user_progress_id": str(user_progress_id),
+                        "scene_id": str(scene_id),
+                        "timestamp": str(datetime.now()),
```

Ruff (0.14.1)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)